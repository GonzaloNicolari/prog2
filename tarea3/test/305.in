# prueba de menores
menores 5
insertar_en_binario (5,g)
menores 5
menores 6
insertar_en_binario (7,k)
menores 5
menores 6
menores 8
insertar_en_binario (4,c)
menores 5	    
menores 6
menores 8
insertar_en_binario (8,i)
insertar_en_binario (8,h)
insertar_en_binario (6,p)
insertar_en_binario (4,m)
imprimir_binario
menores 7
insertar_en_binario (1,e)
menores 6
menores 5
insertar_en_binario (3,b)
insertar_en_binario (9,a)
menores 5
insertar_en_binario (2,d)
imprimir_binario
menores 5
Fin