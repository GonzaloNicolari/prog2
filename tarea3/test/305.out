1>#  prueba de menores.
2>
3>Insertado.
4>
5>
(5,g)
6>Insertado.
7>
8>
(5,g)
9>
-(7,k)
(5,g)
10>Insertado.
11>
(4,c)
12>
(5,g)
-(4,c)
13>
-(7,k)
(5,g)
-(4,c)
14>Insertado.
15>Insertado.
16>Insertado.
17>Insertado.
18>
--(6,p)
---(4,m)
-(7,k)
--(8,i)
---(8,h)
(5,g)
-(4,c)
19>
-(6,p)
--(4,m)
(5,g)
-(4,c)
20>Insertado.
21>
-(4,m)
(5,g)
--(1,e)
-(4,c)
22>
-(4,m)
(1,e)
-(4,c)
23>Insertado.
24>Insertado.
25>
-(4,m)
(1,e)
-(4,c)
--(3,b)
26>Insertado.
27>
--(6,p)
---(4,m)
-(7,k)
--(8,i)
---(8,h)
(5,g)
--(1,e)
---(2,d)
-(4,c)
--(3,b)
---(9,a)
28>
-(4,m)
(1,e)
--(2,d)
-(4,c)
--(3,b)
29>Fin.
